sdk/lib/inflib/infget.c

```diff
@@ -360,10 +364,12 @@ InfpGetLineCount(HINF InfHandle,
 LONG
 InfpGetFieldCount(PINFCONTEXT Context)
 {
-  if (Context == NULL || Context->Line == NULL)
-    return 0;
+  PINFCACHELINE Line;
 
-  return ((PINFCACHELINE)Context->Line)->FieldCount;
+  Line = InfpGetLineForContext(Context);
```

Comment:
You may keep checking for Context == NULL.

Reply:
That's done through InfpGetLineForContext already, no need to replicate the same check throughout all levels.